portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -110,10 +115,12 @@ private String convertToJsonUsingGson(List toConvert) {
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
     String email = userService.getCurrentUser().getEmail();
+    System.out.println(email);
```

Comment:
nit: We should probably take this out? I wonder if we can make use of a logging framework?

Reply:
I originally had that there for testing purposes. Thanks for catching that!